libc/hdr/types/clockid_t.h

```diff
@@ -9,7 +9,7 @@
 #ifndef LLVM_LIBC_HDR_TYPES_CLOCKID_T_H
 #define LLVM_LIBC_HDR_TYPES_CLOCKID_T_H
 
-#ifdef LIBC_FULL_BUILD
+#if defined(LIBC_FULL_BUILD) || defined(_MSC_VER)
```

Comment:
is there a way to do `#if defined(LIBC_FULL_BUILD) || defined(__windows__)`? The `_MSC_VER` macro isn't very clear what it's checking.

Reply:
I will use `_WIN32` then. It is specified in https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170#standard-predefined-macros.